magic-8-ball/03-spin-ai.md

```diff
@@ -173,6 +173,74 @@ id = "magic-8-ball"
 ai_models = ["llama2-chat"]
 ```
 
+## c. Building your Magic 8 Ball application with Python
+
+We need to modify our `magic-eight-ball` component to:
+
+1. Reference a local LLM model
+1. Get a yes/no question from the body of the HTTP request
+1. Use the `Llm.infer` function Spin `Llm` library to generate a response to the question
+
+
+First, update the `handle_request` function to get the question from the request body and return an error if
+the body is empty:
+
+```python
```

Comment:
We should probably add the necessary import(s) here, e.g. `from spin_sdk import llm`.